utils/src/any.rs

```diff
@@ -0,0 +1,30 @@
+/// Provides a best-effort short type name
+pub fn type_name_short<T: ?Sized>() -> &'static str {
+    let s = std::any::type_name::<T>();
+    const SEP: &str = "::";
+    if s.find('<').is_some() {
+        // T is generic so would require more sophisticated processing
+        return s;
+    }
+    match s.rfind(SEP) {
```

Comment:
Perhaps remove the "Flow" suffix too, when present?

Reply:
This is a generic util method, so if done it should be on the `flow_trait` level. And perhaps we can convert the `CamelCaseText` to human readable "Camal case text". Let's leave it for another time.